.github/workflows/run-tests.yml

```diff
@@ -20,12 +21,12 @@ jobs:
           - laravel: 10.*
             testbench: ^8.18
             carbon: ^2.67
-            laravel-package-tools: ^1.16
+            laravel-package-tools: ^1.16.4
```

Comment:
The version in the Laravel repo wasn't compatible with the matrix we're running. Needed a bump to prevent composer errors in CI